cprover_bindings/src/goto_program/expr.rs

```diff
@@ -98,7 +98,11 @@ pub enum ExprValue {
     // {}
     EmptyUnion,
     /// `1.0f`
+    HalfConstant(f16),
```

Comment:
I suggest adding `Float` to the name, e.g. `HalfFloatConstant` or `Float16Constant`.

Reply:
+1 for `Float16Constant`

Reply:
Updating the name to `Float16Constant`